### PR TITLE
Stop using router ports and use webserver port instead

### PR DESCRIPTION
**Pull request by rfay**
Since Gitpod is providing the functionality of ddev-router, let's just not use the router ports, and use the ddev-webserver port explicitly.

<a href="https://gitpod.io/#https://github.com/shaal/ddev-gitpod/pull/51"></a>

### CODE REVIEW

File: .ddev/config.yaml [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
-php_version: "7.3"
+php_version: "7.4"
 webserver_type: nginx-fpm
-router_http_port: "80"
-router_https_port: "443"
-xdebug_enabled: false
-additional_hostnames: []
-additional_fqdns: []
+router_http_port: "8888"
+router_https_port: "8889"
```
Comment (shaal) [inline .ddev/config.yaml:7]: @rfay That is what confused me.

Comment (rfay) [reply .ddev/config.yaml:7]: Those are basically throwaway. We're not using the router. So we're just getting them out of the way.

**Comment by shaal** [comment]
@rfay your thoughts on https://github.com/shaal/ddev-gitpod/pull/51#issuecomment-858943143 ?

**Comment by rfay** [comment]
Sorry, I completely missed your comment in https://github.com/shaal/ddev-gitpod/pull/51#issuecomment-858943143

I don't object in principle, although not sure this is useful locally.

**Comment by rfay** [comment]
https://github.com/shaal/ddev-gitpod/pull/51/commits/2ad3911dd1346122333ef25709749904ccb8d59a puts back the features that make this "normal" on a local machine.

**Comment by rfay** [comment]
Set ports to private by default in https://github.com/shaal/ddev-gitpod/pull/51/commits/bdc12b150e1c403a246e19c929212c6daaca7eeb

File: .gitpod.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ ports:
     visibility: private
   # Direct-connect ddev-webserver port that is the main port
   - port: 8080
-    onOpen: open-preview
```
Comment (rfay) [inline .gitpod.yml]: Hmm... 8080 is the primary use case here. If it's not working we have trouble. It's the only one that really should *not* be ignored.

Comment (shaal) [reply .gitpod.yml]: @rfay 8080 just need a bit more time. When ddev makes it available, the site cannot be shown yet. But with have `gp await port` command that works well. This is all for displaying 8080 preview automatically. The user can always open it manually.

Comment (shaal) [reply .gitpod.yml]: @rfay in the last commit, I restored the port 8080 announcement to the user

**Review by shaal — APPROVED** [review]
Thank you @rfay

**Pull request merged**